keyboards/jlw/vault35_wkl_universal/info.json

```diff
@@ -0,0 +1,769 @@
+{
+    "manufacturer": "jlw",
+    "keyboard_name": "vault 35 WKL universal",
+    "maintainer": "jlw",
+    "bootloader": "stm32-dfu",
+    "bootmagic": {
+        "matrix": [0, 0]
+    },
```

Comment:
```suggestion
```

Reply:
I am not understanding the suggestion here.

Reply:
This is the default functionality. It does not need to be redefined.

Reply:
I see. Corrected